layouts/empty.vue

```diff
@@ -5,5 +5,12 @@
 <script>
 export default {
   name: 'Empty',
+  head() {
+    return {
+      htmlAttrs: {
+        lang: 'fr',
```

Comment:
Je ne sais pas comment on choisir ce code de `lang` du coup ?

Reply:
La justification de ce choix est dans la section « Remarques »

Reply:
Je pense qu'on doit pouvoir indiquer dans le code que cette lang vient de `this.$i18n.fallbackLocale` pour éviter d'avoir une nouvelle string en dur.

Reply:
OK pour moi ✅